CHANGELOG.md

```diff
@@ -3,6 +3,91 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [0.8.0] - 2021-12-09
+### Compatibility Changes:
+- Platform support added:
+  - clang 12 #1385
+  - CUDA 11.4 #1380
+  - GCC 11 #1383
+  - HIP-clang #1338
+  - HIP-nvcc #1337
+  - Xcode 12.5.1 #1385
+  - Xcode 13 #1421
+- Platform support removed:
+  - clang < 5.0 #1385
+  - CUDA < 9.2 #1385
+  - GCC < 7.0 #1385
+  - `hipcc` #1338
```

Comment:
```suggestion
  - hipcc #1338
```
all other compiler names are typeset differently

Reply:
this line must be removed. `hipcc` is still supported but not required anymore. It is now possible to use a modern clang too. `hipcc` is only wrapping the clang compiler and can therefore still be used.

Reply:
Note: We should think about deprecating hipcc. CMake 3.21 introduced native HIP support which only works with hip-clang (hipcc is explicitly not supported).

Reply:
Yes we can do it but not sure if it helps to deprecate something in the changelog only. Normally you add a cmake warning which can be suppressed if needed.